tests/src/test/java/org/neo4j/graphalgo/algo/ShortestPathsProcTest.java

```diff
@@ -123,6 +123,8 @@ public void testResultStream() throws Exception {
             return true;
         });
 
+        System.out.println();
```

Comment:
woops. forget to remove. i was testing load/eval/write progress log and needed a newline ;)